srcs/class/route.cpp

```diff
@@ -1,53 +1,173 @@
 #include "route.hpp"
-#include "../webserv.hpp"
 
-Route::Route( std::string location, std::string root ) {
+// default route
+Route::Route(std::string location, std::string root)
+{
 
 	this->root = root;
 	this->location = location;
```

Comment:
*Tips*: 
 - Préfère préfixer les variables de ta class avec un underscore (`_location` et `_root`)

Et ici tu peux mettre set les valeurs directement dans la déclaration de la fonction!

```suggestion
Route::Route(std::string location, std::string root) : root(root), location(location)
{
```